recipes/progress/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{%set name = "progress" %}
+{%set version = "1.3" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "c88d89ee3bd06716a0b8b5504d9c3bcb3c1c0ab98f96dc7f1dc5f56812a4f60a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+  source_files:
```

Comment:
You do not need to list these files. I think you can safely delete the whole `source_files` section here. Typically you only need this section if there are files in the upstream repo that are not part of the actual package, that might be necessary for testing. :

https://conda.io/docs/building/meta-yaml.html#source-files